collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -465,10 +465,12 @@ public static void moveMediaFiles(String tempMediaPath, File formMediaPath) thro
         }
     }
 
-    public static void saveBitmapToFile(Bitmap bitmap, String path, Bitmap.CompressFormat... format) {
-        Bitmap.CompressFormat compressFormat = Bitmap.CompressFormat.JPEG;
-        if (format[0] != null) {
-            compressFormat = format[0];
+    public static void saveBitmapToFile(Bitmap bitmap, String path) {
+        Bitmap.CompressFormat compressFormat;
+        if (path.endsWith(".png")) {
+            compressFormat = Bitmap.CompressFormat.PNG;
+        } else {
+            compressFormat = Bitmap.CompressFormat.JPEG;
```

Comment:
These 5 lines could be on 1 line, and support mixed case in the file extension.
`final Bitmap.CompressFormat compressFormat = path.toLowerCase.endsWith(".png") ? ...`